docker/docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
       - POSTGRES_DB=evosc_sharp
   
   evosc_sharp:
-    image: evoscsharp:latest
+    image: evoesports/evosc-sharp:latest
```

Comment:
This shouldn't be `evoesports/evosc-sharp` (or atleast not yet). This should be the build of EvoSC-sharp locally, by building the project with `docker build -t evoscsharp .` from the working directory.

Reply:
I see, my apologies, will drop that change

Reply:
No problem haha, it's not documented anywhere, so difficult to know!